habitat-hitl/habitat_hitl/config/test_cfg.yaml

```diff
@@ -0,0 +1,20 @@
+# test_cfg.yaml
```

Comment:
As this is a config for testing only, it should probably live with the tests, not in the library itself. `test/config/habitat_hitl/test_cfg.yaml` would be consistent with the other test configs.